Claudinee11/Property-Pro-Lite1
# server delete and get property

#### What does this PR do?
- adds a endpoints
- adds a deleted property
-add get property

#### Description of Task to be completed
- user-agent can delete and get the property

#### How should this be manually tested?
- clone the repo and switch to the branch ft-deveeloper
- start the server
- use the postman localhost:2000/api/v1/property/ ...` to open get property page

- you should be successfully redirected to the server

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/50420731/61170038-be85e900-a5c8-11e9-83f9-d189795215ec.png)
